src/Request.php

```diff
@@ -153,12 +151,17 @@ protected function throwServerError($body, $httpStatusCode)
             $httpStatusCode);
     }
 
-    protected function getRequestHeaders()
+    protected function getRequestHeaders($contentType = 'application/x-www-form-urlencoded')
```

Comment:
What was the default content type before? if we start passing default content type as above for all apis, will there be any issues for other functionalities?

Reply:
@abhayp, All of other RZP API is accepting POST request with `Content-Type: application/x-www-form-urlencoded` then why payment Links not ?

Reply:
all out apis are json rest apis. we shouldnt ideally support  application/x-www-form-urlencoded for json request. but since legacy code base is like that, we cannot remove it. for new ones we should enforce application/json

Reply:
we can release the new SDK version with `application/json` @ramth05 do you see any challenges or drawbacks?
Anyway we want to move to `application/json`

Reply:
@chirag-patel-rp , don't see any issue as all our API supporting `application/json` and no changes required by enduser . The only thing is that all the API end points we should test it before release new version. and if there is any corner case it should be handled.